shell/gpu/gpu_surface_metal.mm

```diff
@@ -58,6 +58,11 @@
 
   ReleaseUnusedDrawableIfNecessary();
 
+  // When there are platform views in the scene, the drawable needs to be presented in the same
+  // transaction as the one created for platform views. When the drawable are being presented from
+  // the raster thread, there is no such transaction.
+  layer_.get().presentsWithTransaction = [[NSThread currentThread] isMainThread];
```

Comment:
Ideally, I would do the same thing in `gpu_surace_gl` but that does not have access to the layer as it is platform agnostic. The right fix for this is to add a method to the surface delegate that sets the layer property on iOS.